File: java/src/main/java/com/example/tool/ToolB.java
package com.example.tool;

import com.m8test.script.core.api.logger.Logger;

public class ToolB {
    // ToolB调用ToolA的方法
    public void methodB(Logger logger) {
        ToolA toolA = new ToolA();
        toolA.methodA(logger);
        logger.info("Method B in ToolB called.");
    }
}

File: kotlin/src/main/kotlin/com/example/tool/ToolB.kt
package com.example.tool

import com.m8test.script.core.api.console.Console

class ToolB {
    // ToolB调用ToolA的方法
    fun methodB(console: Console) {
        val toolA = ToolA()
        toolA.methodA(console)
        console.log("Method B in ToolB called.")
    }
}